src/styles/main.scss

```diff
@@ -0,0 +1,74 @@
+body {
+  margin: 0;
+  height: 100vh;
+
+  display: grid;
+  place-content: center;
+}
+
+.watch {
+  --size: 80vmin;
+  position: relative;
+
+  height: var(--size);
+  width: var(--size);
+
+  border-radius: 50%;
+  border: calc(var(--size) / 60) dotted black;
+
+  &::before,
+  &::after {
+    content: "";
+    position: absolute;
+    top: 50%;
+    left: 50%;
+
+    transform: translate(-50%, -50%) rotate(-90deg);
+
+    display: block;
+    transform-origin: left;
+  }
+
+  &::before {
+    height: 5px;
+    width: calc(var(--size) / 2);
+
+    background-color: blue;
+
+    animation: 5s forwards move steps(60);
+  }
+
+  &::after {
+    height: 10px;
+    width: calc(var(--size) / 4);
+
+    background-color: green;
+
+    animation: 60s forwards move steps(60);
+  }
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+
+    transform: translate(-50%, -50%);
+
+    width: 20px;
+    height: 20px;
+
+    border-radius: 50%;
```

Comment:
since you use circle twice you can just create a mixin for watch and watch__center

```
@mixin circle($width) {
  width: $width;
  height: $width;
  border-radius: $width/2;
}

.watch {
  @include circle(var(--size));
}
```

Reply:
Done